src/components/VideoListItem/VideoListItem.tsx

```diff
@@ -16,7 +16,7 @@ type VideoListItemProps = {
   title: string;
   duration: number;
   image?: ImageData;
-  seriesId?: string;
+  isSeries?: boolean;
```

Comment:
Same comment about passing in the full media item for VideoListItem (not sure how different it is from Card that it necessitated 2 separate components tbh)

Reply:
Looks like a solution for a responsive tablet page. It is probably have a closer look on it in a separate ticket.